### PR TITLE
feat KHP3-7051: Implement styling for unsettled bill 

### PR DESCRIPTION
**Pull request by its-kios09**
## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Screenshots
improvements

## Related Issue
see https://thepalladiumgroup.atlassian.net/browse/KHP3-7051
*None.*

## Other

*None.*

### CODE REVIEW

**Comment by donaldkibet** [comment]
Thanks @its-kios09 how does this look for imaging and procedure orders

**Comment by its-kios09** [comment]
@donaldkibet I can updated the UI to attain the uniformity

**Comment by ojwanganto** [comment]
Do we still need this? @makombe @its-kios09

File: packages/esm-billing-app/src/billable-services/billiable-item/test-order/test-order-action.scss [hunk 1]
```diff
@@ -0,0 +1,14 @@
+@use '@carbon/type';
+@use '@carbon/layout';
+@use '@carbon/colors';
+
+.actionButton {
+  --cds-layout-size-height-context: var(--cds-layout-size-height-sm, 2rem);
+  --cds-layout-size-height: var(--cds-layout-size-height-context);
```
Comment (donaldkibet) [inline packages/esm-billing-app/src/billable-services/billiable-item/test-order/test-order-action.scss:7]: We should not be directly applying this styles. we should use the type sets [see](https://carbondesignsystem.com/elements/typography/type-sets/)

**Review by donaldkibet — CHANGES_REQUESTED** [review]

**Comment by its-kios09** [comment]
@donaldkibet I think I will revert the scss styling to maintain the uniformity across the procedure, lab and radiology

**Comment by its-kios09** [comment]
Thanks @donaldkibet, I hope the changes haven’t disrupted the layout of the other ESMS. This is what we have collaborated on with the UI/UX design squad from the community.
https://github.com/user-attachments/assets/a43667e7-7c64-487a-9627-d947d8be91cb

**Comment by ojwanganto** [comment]
@donaldkibet @makombe - please help with reviews. @its-kios09 - fix the conflicts

**Comment by its-kios09** [comment]
@ojwanganto sorted

**Review by donaldkibet — CHANGES_REQUESTED** [review]
@its-kios09 how does the button look like on dispensing app before we merge this in

**Comment by its-kios09** [comment]
@donaldkibet It looks the same. The only change I made is aligning the text to the center of the button.

**Review by donaldkibet — APPROVED** [review]

**Pull request merged**